query/parser.go

```diff
@@ -402,9 +364,10 @@ func (p *Parser) insertPropertyKeyValue() {
 	case "to":
 		var unix int64
 		var err error
-		if unix, err = parseDate(value); err == nil {
+		now := time.Now()
+		if unix, err = parseDate(value, now); err == nil {
```

Comment:
This should turn into a single `err != nil` since there's no second `if` anymore.